molecule/verifier/testinfra.py

```diff
@@ -55,7 +55,8 @@ def execute(self):
             self._molecule.verifier_options)
         testinfra_options['env'] = ansible.env
         testinfra_options['debug'] = self._molecule.args.get('debug', False)
-        testinfra_options['sudo'] = self._molecule.args.get('sudo', False)
+        if self._molecule.args.get('sudo'):
+            testinfra_options['sudo'] = True
```

Comment:
I don't believe so, b/c the debug flag is never passed in as an option to testinfra.